$OpenBSD$

OpenBSD does not have RLIMIT_AS. Use RLIMIT_DATA instead.

Index: SAT/MinisatInterfacingNewSimp.hpp
--- SAT/MinisatInterfacingNewSimp.hpp.orig
+++ SAT/MinisatInterfacingNewSimp.hpp
@@ -146,10 +146,10 @@ class MinisatInterfacingNewSimp : public SATSolverWith
       if (max_mem_mb != 0){
           rlim_t new_mem_lim = (rlim_t)max_mem_mb * 1024*1024;
           rlimit rl;
-          getrlimit(RLIMIT_AS, &rl);
+          getrlimit(RLIMIT_DATA, &rl);
           if (rl.rlim_max == RLIM_INFINITY || new_mem_lim < rl.rlim_max){
               rl.rlim_cur = new_mem_lim;
-              if (setrlimit(RLIMIT_AS, &rl) == -1)
+              if (setrlimit(RLIMIT_DATA, &rl) == -1)
                   printf("WARNING! Could not set resource limit: Virtual memory.\n");
           }
       }
